src/client/Microsoft.Identity.Client/PlatformsCommon/Interfaces/IPlatformProxy.cs

```diff
@@ -90,7 +90,7 @@ internal interface IPlatformProxy
         /// <summary>
         /// Go to a URL using the OS default browser. 
         /// </summary>
-        Task StartDefaultOsBrowserAsync(string url);
```

Comment:
Do we really need the param to have a default value? I find it easier to refactor code / reason about code when I'm asked to provide all params.

Reply:
Removed the default value